core/dbt/adapters/README.md

```diff
@@ -1 +1,30 @@
 # Adapters README
+
+The Adapters module is responsible for defining database connection methods, schema for caching, how relations are defined, and the two major types of connection types we have base and sql.
```

Comment:
I don't think that base and sql are "connection types" per se. I'd describe them as a base implementation and a sql implementation of a connection.

Reply:
These are all great insights, will pull them in and update for next iteration! thanks for the eyes one @gshank